src/WindowsAzureMessaging/WindowsAzureMessaging/SBConnectionString.m

```diff
@@ -7,61 +7,48 @@
 
 @implementation SBConnectionString
 
-NSString* const defaultListenSasRuleName = @"DefaultListenSharedAccessSignature";
-NSString* const defaultFullSasRuleName = @"DefaultFullSharedAccessSignature";
-
-
-+ (NSString*) stringWithEndpoint:(NSURL*)endpoint issuer:(NSString*) issuer issuerSecret:(NSString*)secret
-{
-    if(!endpoint || !issuer || !secret)
-    {
-        NSLog(@"endpoint/issuer/secret can't be null.");
-        return nil;
-    }
-    
-    endpoint = [SBNotificationHubHelper modifyEndpoint:endpoint scheme:@"sb"];
-    
-    NSString* endpointUri = [endpoint absoluteString];
-    if([[endpointUri lowercaseString] hasPrefix:@"endpoint="])
-    {
-        return [NSString stringWithFormat:@"%@;SharedSecretIssuer=%@;SharedSecretValue=%@",endpointUri,issuer,secret];
-    }
-    else
-    {
-        return [NSString stringWithFormat:@"Endpoint=%@;SharedSecretIssuer=%@;SharedSecretValue=%@",endpointUri,issuer,secret];
-    }
+NSString *const defaultListenSasRuleName = @"DefaultListenSharedAccessSignature";
```

Comment:
IMO this is not the PR for reformatting all of the existing code.

Reply:
Fair, just using some clang formatting to clean up existing code to standardize